spec/lib/whitehall_importer/create_image_revision_spec.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+RSpec.describe WhitehallImporter::CreateImageRevision do
+  describe "#call" do
+    let(:img_url) { "https://assets.publishing.service.gov.uk/government/uploads/valid-image.jpg" }
+    let(:img_file) { "960x640.jpg" }
+    let(:whitehall_image) do
+      { "url": img_url, "caption": "Caption", "alt_text": "Alt text" }.with_indifferent_access
+    end
+
+    before do
+      stub_request(:get, img_url).to_return(
+        status: 200,
+        body: file_fixture(img_file).read,
+      )
+    end
```

Comment:
I've found this a bit surprising as we then have a similar `stub_request` in the before of the next context.

Though I've made a suggestion in the `create_image` method of the other test file that could resolve this.